app/views/shared/customer_support/_live_chat_scripts.html.erb

```diff
@@ -139,4 +139,16 @@
   }
 
   const customPlugin = CXBus.registerPlugin('Custom');
+
+  <% if EnrollRegistry.feature_enabled?(:live_chat_widget_error_logging) %>
+    // throw error when the webchat gets rejected or has an error for visibility
+
+    customPlugin.subscribe('WebChat.rejected', function(e){
+      (console.error || console.log).call(console, e.stack || e);
```

Comment:
why not just `console.error(e)`?

Reply:
muscle memory from having to support IE9 where error.stack was not a thing. I can take it out.